v2/tools/generator/internal/codegen/testdata/ArmResource/Arm_oneof_resource_conversion_on_arm_type_only_azure.golden

```diff
@@ -196,7 +196,16 @@ func (resource *FakeResource) updateValidations() []func(old runtime.Object) err
 		func(old runtime.Object) error {
 			return resource.validateResourceReferences()
 		},
+		resource.validateImmutableProperties}
+}
+
+// validateImmutableProperties validates all immutable properties
+func (resource *FakeResource) validateImmutableProperties(old runtime.Object) error {
```

Comment:
We suppress some pipeline stages for these golden tests as they focus on property structure and the functions are noise - perhaps validation should be suppressed too? #unsure

Reply:
We already have `validateResourceReferences` in the golden tests though and its been a while..

Reply:
We already have validateResourceReferences in the golden tests though